lib/components/viewers/viewers.css

```diff
@@ -246,7 +301,7 @@
 .otp .stop-viewer .stop-viewer-body .next-arrival-row .next-arrival-label {
   align-items: center;
   display: flex;
-  max-width: 250px;
+  max-width: 75%;
```

Comment:
Percentages on text can be wonky especially on strange screen widths. Can we do this with grid instead?